frontend/templates/product/sections/ProductReviewsSection/index.tsx

```diff
@@ -217,15 +219,41 @@ function ReviewsStats({
 
 type ProductReviewLineItemProps = {
    review: ProductReview;
+   variantSku: string | undefined | null;
 };
-function ProductReviewLineItem({ review }: ProductReviewLineItemProps) {
+function ProductReviewLineItem({
+   review,
+   variantSku,
+}: ProductReviewLineItemProps) {
+   const isAdminUser = useAuthenticatedUser().data?.isAdmin ?? false;
+   const appContext = useAppContext();
+
    return (
       <Box
          py="6"
          borderBottomWidth="1px"
          borderColor="gray.200"
          data-testid="product-review-line-item"
       >
+         {isAdminUser && variantSku && (
+            <Button
+               variant="link"
+               as={Link}
+               bgColor="transparent"
+               textColor="brand"
+               fontFamily="heading"
+               lineHeight="1.29"
+               fontWeight="semibold"
+               fontSize="14px"
+               color="brand.500"
+               textAlign="center"
+               paddingBottom="10px"
+               href={`${appContext.ifixitOrigin}/User/Reviews/${variantSku}/userid=${review.author?.userid}`}
```

Comment:
```suggestion
               href={`${appContext.ifixitOrigin}/User/Reviews/${variantSku}?userid=${review.author?.userid}`}
```

Reply:
Thanks for pointing that out! [2efb3ba](https://github.com/iFixit/react-commerce/pull/1573/commits/2efb3ba69edb007c383f0ea40e10b52a74c6aed3)